lib/bson/error/invalid_dbref_argument.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module BSON
+  class Error
+    # Raised when trying to create a BSON::DBRef from an object that is an invalid DBRef.
+    #
+    # @api private
```

Comment:
Exception classes are not private, only their constructors are.

Reply:
done

Reply:
There are 2 or 3 other classes that this applies to.

Reply:
done